content/messaging/customer-chat-api/index.mdx

```diff
@@ -2415,6 +2415,44 @@ Here's what you need to know about **webhooks**:
 
 You can create and manage webhooks via the [Configuration API](/management/configuration-api/#webhooks). Refer to this document to see the list of **available webhooks**, as well as **sample payloads**.
 
+# Errors
+
+<CodeResponse title={'Format of error payload'}>
+
+```json
+{
+  "error": {
+    "type": "misdirected_request",
+    "message": "Wrong region",
+    "data": {
+      // optional
```

Comment:
maybe this comment should be in line 2427? after all whole `data` object is optional